bin/zerobufCxx.py

```diff
@@ -15,6 +15,8 @@
 import sys
 from collections import namedtuple, OrderedDict
 
+inline_implementation = False
```

Comment:
Removed global default value of inline_implementation and global keyword. Only needed read access to variable.